src/renderer/scss/component/_snack-bar.scss

```diff
@@ -1,30 +1,54 @@
 .snack-bar {
-  padding: $spacing-vertical;
-  position: fixed;
-  top: $spacing-vertical;
+  background-color: var(--color-primary);
+  border-radius: 10px;
+  box-shadow: var(--box-shadow-layer);
+  color: #f0f0f0;
+  display: flex;
+  justify-content: space-between;
   left: 0;
-  right: 0;
   margin-left: auto;
   margin-right: auto;
-  min-width: 300px;
   max-width: var(--snack-bar-width);
-  background: var(--color-dark-overlay);
-  color: #f0f0f0;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  border-radius: 2px;
+  min-width: 300px;
+  opacity: 0.95;
+  padding: 14px 20px 10px 20px;
+  position: fixed;
+  right: 0;
+  top: $spacing-vertical;
   transition: all var(--transition-duration) var(--transition-type);
+  width: 100%;
   z-index: 10000; /*hack to get it over react modal */
-}
 
-.snack-bar__action {
-  display: inline-block;
-  text-transform: uppercase;
-  color: var(--color-primary-light);
-  margin: 0px 0px 0px $spacing-vertical;
-  min-width: min-content;
-  &:hover {
-    text-decoration: underline;
+  .message {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+
+    div {
+      &:nth-of-type(1) {
+        font-size: 30px;
+        margin-right: 20px;
+      }
+      &:nth-of-type(2) {
+        font-size: 16px;
+        margin-bottom: 4px;
+      }
+    }
+  }
+
+  .snack-bar__action {
+    color: var(--color-primary-light);
+    display: inline-block;
+    margin-bottom: 4px;
+    min-width: min-content;
+    text-transform: uppercase;
+
+    span {
+      color: #f0f0f0;
```

Comment:
This should just be a color variable. It's just the same as the color you have above so maybe this isn't needed?